cogboard-webapp/src/components/TabPanel.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
Pls import PropTypes props separately for consistency with other recently added component files